app/models/project.rb

```diff
@@ -1,97 +1,22 @@
 # frozen_string_literal: true
-class Project
```

Comment:
The `Project` class is now an ApplicationRecord class and the previous project functionality that interacts with Mediaflux has been moved to `ProjectMediaflux`